emails/templates/ng_alert_notification.html

```diff
@@ -101,7 +116,7 @@
 [[ end ]]
 
 [[ if gt (len .Message) 0 ]]
- [[ .Message ]]
+  <span style="white-space: pre-line;">[[ .Message ]]</span>
```

Comment:
This is the only real change that relates to this PR. The rest of the changes are recreations of past PRs, in the generator.